src/collections/_documentation/platforms/node/azure_functions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Azure Functions
+sidebar_order: 9
+---
+
+Install our Nodejs SDK in Azure:
+```basic
+ npm install @sentry/node
+```
+
+Set up Sentry error logging for your azure function:
+```jsx
+const Sentry = require("@sentry/node");
+
+Sentry.init({
+  dsn: "<Your dsn>",
+});
+
+module.exports = async function (context, req) {
+    try {
+        callUndefinedFunction(); // Call undefined function.
+    } catch (error) {
+        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
+        await Sentry.flush(2000);
+    }
+
+    context.res = {
+        status: 200, /* Defaults to 200 */
+        body: "Hello from Azure cloud function"
+    };
+}
+```
```

Comment:
```suggestion
```js
const Sentry = require("@sentry/node");

Sentry.init({
    dsn: "___PUBLIC_DSN___"
});

module.exports = async function (context, req) {
    try {
        await callUndefinedFunction(); // Call undefined function.
    } catch (error) {
        Sentry.captureException(error); // Capture the exception in Sentry dashboard.
        await Sentry.flush(2000);
    }

    context.res = {
        status: 200, /* Defaults to 200 */
        body: "Hello from Azure cloud function"
    };
}
```
```